src/lib/y2partitioner/widgets/configurable_blk_devices_table.rb

```diff
@@ -152,12 +152,7 @@ def show_columns(*column_names)
       # @macro seeAbstractWidget
       # @see #columns_help
       def help
-        header = _(
-          "<p>This view shows storage devices.</p>" \
-          "<p>The overview contains:</p>" \
-        )
-
-        header + columns_help
+        "<p>Table containing the list of selected devices, where:</p>" + columns_help
```

Comment:
Not sure if "list of selected devices" is really precise and understandable here.

Reply:
Yes, I am not sure either. Do you have any idea to improve it?